packages/@react-aria/focus/stories/FocusScope.stories.tsx

```diff
@@ -92,12 +94,12 @@ function Example({usePortal, contain}: StoryProps) {
 
   return (
     <div>
-      <input />
+      <input  />
 
       <button type="button" onClick={() => setOpen(true)}>
         Open dialog
       </button>
-      <input />
+
```

Comment:
Mind keeping this input here? It is helpful for testing the "contain=false" case, specifically for testing that tabbing forward through the dialog should land you on the element after the trigger